src/background/services/wallet/handlers/models.ts

```diff
@@ -23,6 +23,7 @@ export type ImportLedgerWalletParams = {
   pubKeys?: PubKeyType[];
   secretType: SecretType.Ledger | SecretType.LedgerLive;
   name?: string;
+  onlyCheckWalletIsExist?: boolean;
```

Comment:
Should we call this `dryRun` instead? 
```suggestion
  dryRun?: boolean;
```